src/Business/Detection/Common/Commands/ICommand.cs

```diff
@@ -1,4 +1,4 @@
-namespace Business.Detection.Fetching.Commands
+namespace Business.Detection.Common.Commands
```

Comment:
As far as this command is gonna be used for any further features we are going to have, won't be better to move it under the root `Business` project (I mean at the root level of the `Business` project)?

Reply:
Done.